src/arrays/Arrays.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace arrays;
+
+use arrays\ArraysInterface;
+
+class Arrays implements ArraysInterface
+{
+    /**
+     * @param array $input
+     * @return array
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $result = [];
+        $arrayCount = count($input);
+
+        for ($i = 0; $i < $arrayCount; $i++) {
```

Comment:
could you please try to use php array functions?

Reply:
done